lessons/bn/ecto/changesets.md

```diff
@@ -0,0 +1,291 @@
+%{
+  version: "1.2.2",
+  title: "চেইঞ্জসেট",
+  excerpt: """
+  ডাটাবেসে ডাটা, ইনসার্ট, আপডেট অথবা ডিলেট করতে ব্যবহৃত `Ecto.Repo.insert/2`, `update/2` এবং `delete/2` ফাংশনগুলোর প্রথম প্যরামিটার হিসেবে চেইঞ্জসেট লাগে, কিন্তু কি এই চেইঞ্জসেট?
+  
+  ইনপুট ডাটা তে কোনো এরর আছে কি না এটা পরীক্ষা করে দেখা প্রত্যেক ডেভেলপারের নিত্যদিনের কাজের অংশ — আমাদের ডাটাগুলো ব্যবহারের আগে আমরা ডাটাগুলো সঠিক অবস্থায় আছে তা নিশ্চিত করতে চাই।
+
+  এক্টো `Changeset` মডিউল এবং ডাটা স্ট্রাকচারের মাধ্যমে এক্টো আমাদেরকে চেইঞ্জিং ডাটা নিয়ে কাজ করার একটা পরিপূর্ণ সমাধান প্রদান করে।   
+  এই অধ্যায়ে আমরা এই ফাংশনালিটিটি দেখবো এবং কিভাবে ডাটাবেসে ডাটা রাখার আগে, ডাটা ইন্টেগ্রিটি যাচাই করতে হয় তা শিখবো। 
+  """
+}
+---
+
+## প্রথম চেইঞ্জসেট তৈরি
+
+চলুন, একটা ফাঁকা চেইঞ্জসেট স্ট্রাক্ট `%Changeset{}` দেখা যাক:
```

Comment:
@wasi0013  bhai, "চেইঞ্জ" or "চেঞ্জ" which one should we use?

Reply:
Well, Change has a long `a` sound in it so, I've added `ই` in the middle. 
`চেঞ্জ` will save us some keystrokes though. Your thoughts? :)

Reply:
@wasi0013 bhai, I'm clueless. Let's try to figure out which one is more commonly used. :D

Reply:
So far, I can see that "চেইঞ্জ" is more commonly used.

Reply:
Think we can live with that for now. :)